apps/web/src/pages/Pools/AssessmentPlanBuilderPage/AssessmentPlanBuilderPage.tsx

```diff
@@ -231,28 +232,23 @@ export const AssessmentPlanBuilderPage = () => {
 
   // Note: Should technically be in subNav of layout?
```

Comment:
@esizer I updated this but `navigationCrumbs` is an unused variable. does it need to remain in the code or can it be removed entirely?

Reply:
Yeah, you can :axe: this. Those crumbs are defined in the layout component.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/aed46eb6ef5113587503145445582f08de28b495/apps/web/src/utils/poolUtils.tsx#L317-L347

Reply:
[b463e04](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9796/commits/b463e0473b4a2c21f01af4b6ece2af259d4ab6c0)